src/internal/operations/backup.go

```diff
@@ -117,9 +117,26 @@ func (op BackupOperation) validate() error {
 // pointer wrapping the values, while those values
 // get populated asynchronously.
 type backupStats struct {
-	k             *kopia.BackupStats
-	ctrl          *data.CollectionStats
-	resourceCount int
+	k              *kopia.BackupStats
+	ctrl           *data.CollectionStats
+	resourceCount  int
+	assistBackupOp bool
+}
+
+// To qualify as an assist backup op, all of the following must be true:
+// 1. new deets were produced during this operation
+// 2. we have a valid snapshot ID
+// 3. we don't have any non-recoverable errors
+// 4. we recorded recoverable errors
+func isAssistBackupOp(
```

Comment:
Nit, I think adding `op` to the name adds confusion.  There's no such thing as an `assist operation`.  There's only an assist-category backup at the end of a backup operation.

```suggestion
func isAssistBackup(
```